src/test/groovy/graphql/execution/reactive/tck/CompletionStageMappingOrderedPublisherRandomCompleteTckVerificationTest.java

```diff
@@ -0,0 +1,71 @@
+package graphql.execution.reactive.tck;
```

Comment:
I have made a series of new TestNG compliance tests that use the TCK to ensure they are valid publishers